cms/models.py

```diff
@@ -1248,11 +1248,19 @@ def parent(self):
         return self.get_parent().specific
 
     def get_context(self, request, *args, **kwargs):
+        preview_context = {}
+
+        if request.is_preview:
+            preview_context = {
```

Comment:
@ahmed-arbisoft I have added a `preview_context` here that will render the certificate view version while clicking on `PREVIEW` button. 
Ref: 

<img width="1120" alt="Screen Shot 2019-08-01 at 4 57 38 PM" src="https://user-images.githubusercontent.com/7334669/62291490-806c3e80-b47d-11e9-93db-13976a6ea316.png">